lib/active_model/serializer/adapter/json.rb

```diff
@@ -37,9 +37,7 @@ def serializable_hash(options = {})
             @result = @core.merge @hash
           end
 
-          if root = options.fetch(:root, serializer.json_key)
-            @result = { root => @result }
-          end
+          @result = { root => @result } if root
           @result
```

Comment:
Simple housekeeping question (not a blocker or nor requires any serious thought):

Is the code easier to read as

```ruby
@result = { root => @result } if root
@result
```

or

```ruby
if root
  @result = { root => @result }
else
  @result
end
```